From: Modestas Vainius <modestas@vainius.eu>
Subject: [PATCH] Disable QtScriptBindings check

QtScriptBindings is a runtime dependency which is not
needed at build time. Disable cmake check.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -79,9 +79,6 @@ if( WITH_PLAYER )
 
     macro_log_feature( KDE4_FOUND "kdelibs" "The toolkit Amarok uses to build" "http://www.kde.org" TRUE "4.3.0" "" )
 
-    find_package( QtScriptQtBindings REQUIRED )
-    macro_log_feature( QTSCRIPTQTBINDINGS_FOUND "qtscript-qt" "QtScript Qt Bindings" "http://code.google.com/p/qtscriptgenerator/" TRUE "" "" )
-
     add_definitions(${QT_DEFINITIONS} ${KDE4_DEFINITIONS})
 
     find_package(MySQLAmarok REQUIRED)
